ui/lib/core/addon/templates/components/replication-action-demote.hbs

```diff
@@ -14,7 +14,7 @@
         @message="Demoting this DR primary cluster
           would result in a DR secondary and in that mode Vault is read-only. This
           cluster is also currently operating as a Performance
-          capitalize model.performance.modeForUrl}}, demoting it will leave your
+          capitalize {{model.performance.modeForUrl}}, demoting it will leave your
```

Comment:
The `capitalize` part is a helper (so we're going for "Perforrmance Primary" or "Performance Secondary"), so we should move that inside too

```suggestion
          {{capitalize model.performance.modeForUrl}}, demoting it will leave your
```

Reply:
When I read that message I was confused.  Thanks!  Makes a lot more sense now.

Reply:
Looks like this with the correct fix. 
![image](https://user-images.githubusercontent.com/6618863/76646738-20fd0380-6521-11ea-921a-3d0cf13e2c08.png)